### PR TITLE
AGIOS-602 Rename AeroGear Push Objective-C project

### PR DESCRIPTION
**Pull request by jcesarmobile**

### CODE REVIEW

**Comment by jcesarmobile** [comment]
@danielpassos I can't add you as reviewer, can you add yourself? and probably give me permissions on the repo

File: AeroGear-Push.podspec [hunk 1]
```diff
@@ -1,13 +1,13 @@
 Pod::Spec.new do |s|
-  s.name         = "AeroGear-Push"
+  s.name         = "AeroGearPush"
   s.version      = "1.2.0"
   s.summary      = "AeroGear UnifiedPush Client Registration SDK."
-  s.homepage     = "https://github.com/aerogear/aerogear-ios-push"
+  s.homepage     = "https://github.com/jcesarmobile/aerogear-ios-push-objc"
```
Comment (danielpassos) [inline AeroGear-Push.podspec]: jcesarmobile? :grimacing:

File: AeroGear-Push.podspec [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
   s.license      = 'Apache License, Version 2.0'
   s.author       = "Red Hat, Inc."
-  s.source       = { :git => 'https://github.com/aerogear/aerogear-ios-push.git', :tag => '1.2.0' }
+  s.source       = { :git => 'https://github.com/jcesarmobile/aerogear-ios-push-objc.git', :tag => s.version }
```
Comment (danielpassos) [inline AeroGear-Push.podspec]: jcesarmobile? :grimacing:

Comment (jcesarmobile) [reply AeroGear-Push.podspec]: yeah, I declare ownership now 😛 

Fixed

**Comment by jcesarmobile** [comment]
Rebased after merging your PR as it had conflicts on the README.md

**Pull request merged**